WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -772,16 +768,15 @@ class PagesViewModel
 
     private fun handleInvalidateUploadStatus(ids: List<LocalId>) {
         launch {
-            _invalidateUploadStatus.postValue(ids)
-            invalidatePages()
+            _invalidateUploadStatus.value = ids
+            refreshPages()
```

Comment:
This line will keep hitting the database on each media upload progress state change, which isn't optimal. But I didn't come up with a different reliable solution. I haven't noticed any performance side-effects, so I guess we can keep it unless you can think of a better solution.

Reply:
Yes, we will have to keep it based on my observation.